.github/workflows/build.yml

```diff
@@ -90,7 +88,7 @@ jobs:
         run: .github/ci.sh test_dist
 
       - if: >-
-          (startsWith(github.ref, 'refs/tags/v')
+          (startsWith(github.head_ref, 'releases-')
```

Comment:
Should this be `release` instead of `releases`?

Reply:
Yup! good catch. Fixed